OloEngine/src/OloEngine/Audio/SoundGraph/SoundGraphSound.cpp

```
@@ -503,31 +523,56 @@ namespace OloEngine::Audio::SoundGraph
+    /// detached source, or no engine spatializer), so the stored value is what a later
+    /// registration / the getters observe.
 
     void SoundGraphSound::SetLocation(const glm::vec3& location)
     {
         OLO_PROFILE_FUNCTION();
         m_Position = location;
+        SyncSpatialPositionToSource();
     }
 
     void SoundGraphSound::SetVelocity(const glm::vec3& velocity)
     {
```

Cppcheck (2.21.0)

[style] The function 'ProcessThunk' is never used.

(unusedFunction)